### PR TITLE
Updated the Docker quickstart to improve the Portus section

### PR DESCRIPTION
**Pull request by mssola**
The Portus section has been improved in order to contain all the latest
documentation. This way, customers will be able to have Portus up and ready for
usage.

Ping @eotchi, @taroth21, @flavio and @jordimassaguerpla

Signed-off-by: Miquel Sabaté Solà <msabate@suse.com>

### CODE REVIEW

File: xml/vt_docker_quick.xml [hunk 1]
```diff
@@ -823,9 +823,51 @@ RUN zypper -n in vim</screen>
   </para>
 
   <para>
+    Portus is accessible for &slea;&nbsp;12 customers through the Containers
+    module. This means that you should perform the following command to install
```
Comment (taroth21) [inline xml/vt_docker_quick.xml]: To install Portus, use the following command:

File: xml/vt_docker_quick.xml [hunk 2]
```diff
@@ -823,9 +823,51 @@ RUN zypper -n in vim</screen>
...
+    Portus:
+  </para>
+
+  <screen>sudo zypper in Portus</screen>
+
+  <para>
+    After doing this, you should proceed on configuring your freshly installed
```
Comment (taroth21) [inline xml/vt_docker_quick.xml]: Here it is not clear to me if this belongs to the step before (which also deals with configuration)? Maybe you can specify in the step above which sections of the external references there user is supposed to read and execute ( probably 'Portus’ dependencies' and 'Initial setup')?

File: xml/vt_docker_quick.xml [hunk 3]
```diff
@@ -823,9 +823,51 @@ RUN zypper -n in vim</screen>
...
+    Portus instance. This is thoroughly documented here:
+    <link xlink:href="http://port.us.org/docs/setups/1_rpm_packages.html#portus-dependencies"/>.
+    This document will help you to get through the installation process, and
+    it will also warn you about some of the common pitfalls.
+  </para>
+
+  <para>
```
Comment (taroth21) [inline xml/vt_docker_quick.xml]: Here it is not clear to me if this belongs to the step before (which also deals with configuration)? Maybe you can specify in the step above which sections of the external references there user is supposed to read and execute ( probably 'Portus’ dependencies' and 'Initial setup')?

File: xml/vt_docker_quick.xml [hunk 4]
```diff
@@ -823,9 +823,51 @@ RUN zypper -n in vim</screen>
...
+    After installing Portus, you might want to configure it. Note that the
+    portusctl command has already done most of the needed configuration.
+    However, there are some configurable values that you should have to take a
+    look before using Portus. This is thoroughly explained in this
+    documentation page:
+    <link xlink:href="http://port.us.org/docs/Configuring-Portus.html"/>.
+  </para>
+
+  <para>
```
Comment (taroth21) [inline xml/vt_docker_quick.xml]:
```[...]
 <step>
   <para>To apply the configuration changes, restart Apache (this is required after each configuration change).
 </para> 
</step>

File: xml/vt_docker_quick.xml [hunk 5]
```diff
@@ -823,9 +823,51 @@ RUN zypper -n in vim</screen>
...
+    After doing this, you should restart Apache so Portus fetches what you have
+    changed in the configuration. Note that you will always have to restart
+    Apache after changing a configuration value.
+  </para>
+
+  <para>
+    Finally, when entering Portus for the first time, you will be required
+    to enter some information about your installed registry. We have written
+    down the process about this in the following section of our documentation:
```
Comment (taroth21) [inline xml/vt_docker_quick.xml]: For details, see <link xlink:href="http://port.us.org/docs/setups/1_rpm_packages#the-default-installation.html"/>.

File: xml/vt_docker_quick.xml [hunk 6]
```diff
@@ -823,9 +823,51 @@ RUN zypper -n in vim</screen>
...
+    <link xlink:href="http://port.us.org/docs/setups/1_rpm_packages#the-default-installation.html"/>.
+  </para>
+
+  <para>
+    At this point, you should be able to enjoy Portus!
+  </para>
```
Comment (taroth21) [inline xml/vt_docker_quick.xml]: Maybe rephrase to: The Portus setup is now complete and you can start using Portus.

**Comment by mssola** [comment]
@taroth21 fixed

**Comment by mssola** [comment]
@taroth21 fixed

**Comment by taroth21** [comment]
@mssola: Thanks a lot, merged into develop now!
